lib/lita/handlers/versioner.rb

```diff
@@ -109,14 +109,19 @@ def github_handler(request, response)
         when "pull_request"
           # https://developer.github.com/v3/activity/events/types/#pullrequestevent
           # If the pull request is merged with some commits
-          if payload["action"] == "closed" && payload["pull_request"]["merged"]
-            new_version = bump_version_in_git
-            trigger_build("v#{new_version}")
+          if payload["action"] == "closed"
+            if payload["pull_request"]["merged"]
+              inform("'#{payload["pull_request"]["html_url"]}' is just merged. Working on it...")
```

Comment:
Oops! Sorry, didn't mean to have you add another inform here. This will cause us to spit out two messages per build ("Working on it" and then "started build") ... one is probably enough. Sorry to randomize.